models/Body.js

```diff
@@ -37,25 +36,23 @@ const Body = sequelize.define('body', {
     },
     email: {
         type: Sequelize.STRING,
-        allowNull: false,
+        allowNull: true,
         validate: {
-            notEmpty: { msg: 'Email must be set.' },
+            isValid(value) {
+                if ((value === null || value === undefined) && this.status !== 'deleted') {
+                    throw new Error('Email must be set.');
+                }
+            },
             isEmail: { msg: 'Email must be valid.' }
         }
     },
     phone: {
         type: Sequelize.STRING,
-        allowNull: true,
-        validate: {
-            notEmpty: { msg: 'Phone must be set.' },
-        }
+        allowNull: true
     },
     address: {
         type: Sequelize.TEXT,
-        allowNull: true,
-        validate: {
-            notEmpty: { msg: 'Address must be set.' },
```

Comment:
I thought this prevented it from having empty addresses in the frontend but I just realised it might also be due to an empty string being set as default in the frontend. Therefore this PR is in draft